docs/docs/pixabay-source-plugin-tutorial.md

```diff
@@ -377,7 +377,7 @@ You've written a local Gatsby plugin that:
 
 Congratulations!
 
-## Where next?
+## What's next?
```

Comment:
Please revert this, too, as "Where next?" makes more sense here (saying "where can I look to go further"?)